crates/bevy_render/src/camera/camera.rs

```diff
@@ -333,6 +338,15 @@ impl RenderTarget {
             RenderTarget::Image(image_handle) => {
                 images.get(image_handle).map(|image| &image.texture_view)
             }
+            RenderTarget::InitialWindow => {
+                let window_id = windows
+                    .initial
+                    .expect("expected an initial window to exist if going to retrieve view");
```

Comment:
```suggestion
                    .unwrap();
```
I don't think this error message is clearer than the default unwrap message.